src/lottie/lottiemodel.h

```diff
@@ -539,8 +539,8 @@ class Composition : public Object {
     {
         return long(frameAtPos(timeInSec / duration()));
     }
-    size_t totalFrame() const { return mEndFrame - mStartFrame; }
-    long   frameDuration() const { return mEndFrame - mStartFrame - 1; }
+    size_t totalFrame() const { return mEndFrame - mStartFrame + 2; }
+    long   frameDuration() const { return mEndFrame - mStartFrame; }
```

Comment:
I also searched for code related to this issue in android implement.
and I found the code for `-0.1f`. (https://github.com/airbnb/lottie-android/blob/456332497619eee69fcf82a8005ed58c95761a5c/lottie/src/main/java/com/airbnb/lottie/parser/LottieCompositionMoshiParser.java#L69 )

And I also found implementations for `duration`.
(https://github.com/airbnb/lottie-android/blob/da7d284002fccd4226877ea3c4107e2c7246f107/lottie/src/main/java/com/airbnb/lottie/LottieComposition.java#L216)

But I couldn't find it for totalFrame. 
I'm not sure if adding 2 frame to totalFrame will cover all scenarios.
but when I tested on a few resources there were no problems.

What do you think? @hermet @smohantty